doc/source/whatsnew/v2.0.0.rst

```diff
@@ -1362,7 +1362,7 @@ Groupby/resample/rolling
 - Bug in :meth:`.DataFrameGroupBy.describe` produced incorrect results when data had duplicate columns (:issue:`50806`)
 - Bug in :meth:`.DataFrameGroupBy.agg` with ``engine="numba"`` failing to respect ``as_index=False`` (:issue:`51228`)
 - Bug in :meth:`DataFrameGroupBy.agg`, :meth:`SeriesGroupBy.agg`, and :meth:`Resampler.agg` would ignore arguments when passed a list of functions (:issue:`50863`)
--
+- Bug in :meth:`DataFrameGroupBy.agg` with multiple groupings would not include groupings in the result when they occurred in the selected columns (:issue:`51186`)
```

Comment:
> with multiple groupings

this might not be clear without looking at the new test

Reply:
Yea, I wasn't sure how to describe that the bug only occurs with `.groupby([x, y])` and not `.groupby(x)`. Is there better language that could be used here, or maybe just remove it altogether?

Reply:
'after subsetting columns (e.g. `gb[["a", "b"]].agg(...)`'?